lapis2/src/main/kotlin/org/genspectrum/lapis/model/SiloFilterExpressionMapper.kt

```diff
@@ -62,6 +75,16 @@ class SiloFilterExpressionMapper(private val allowedSequenceFilterFields: Sequen
         return Pair(filterExpressionId, type)
     }
 
+    private fun mapToVariantQueryFilter(variantQuery: String): SiloFilterExpression {
+        if (variantQuery.isBlank()) {
+            throw IllegalArgumentException(
+                "variantQuery cannot be empty",
+            )
+        }
+
+        return VariantQueryFacade().map(variantQuery)
```

Comment:
Make this a field and let the DI inject a `VariantQueryFacade` here instead of creating a new instance every time.